docs/tutorials/projects/integrating-viam-with-openai.md

```diff
@@ -266,31 +250,30 @@ Click **Save config** to finish adding the detector.
 With the rover and tutorial code set up and it is time to bring your companion robot to life!
 Let's call her "Rosey", and bring her to life by running:
 
-```bash
+``` bash
 ./run.sh
 ```
 
 Now, you can start talking to Rosey.
 {{<imgproc src="/tutorials/ai-integration/rosey_robot.jpg" resize="400x" declaredimensions=true alt="Viam Rover Rosey." class="alignright" style="max-width: 350px">}}
 Any time she hears the keyword "Rosey", she will pay attention to anything you say immediately afterwards.
-For example, if you say _"Hello Rosey, what do you think will happen today?"_, the phrase _"what do you think will happen today"_ will be sent to OpenAI's chat completion API, and you'll get a response back similar to _"It is impossible to predict what will happen today.
-Every day is different and unpredictable!"_
+For example, if you say *"Hello Rosey, what do you think will happen today?"*, the phrase *"what do you think will happen today"* will be sent to OpenAI's chat completion API, and you'll get a response back similar to *"It is impossible to predict what will happen today.
```

Comment:
yeah bit concerning it's changing this to stars because that's now not how it's supposed to be configured